htdocs/core/class/html.formticket.class.php

```diff
@@ -49,10 +49,15 @@ class FormTicket
 	public $db;
 
 	/**
-	 * @var string	The track_id of the ticket. Used also for the $keytoavoidconflict to name session vars to upload files.
+	 * @deprecated
```

Comment:
No the comment was good.
track_id is not trackid.
They are 2 different fields used for 2 different purposes.